util/src/journaldb/overlayrecentdb.rs

```diff
@@ -191,6 +191,13 @@ impl OverlayRecentDB {
 		trace!("Recovered {} overlay entries, {} journal entries", count, journal.len());
 		JournalOverlay { backing_overlay: overlay, journal: journal, latest_era: latest_era }
 	}
+
+	#[inline]
+	fn to_short_key(key: &H256) -> H256 {
```

Comment:
other DBs don't have in-memory overlay for past blocks and don't require such conversion